controllers/keywordsController.js

```diff
@@ -22,4 +22,52 @@ const list = async (req, res) => {
   }
 };
 
-module.exports = { list };
+const put = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isEmptyString(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
+  }
+
+  if (
+    !isValidArray(req.body.includedKeyword) ||
+    !req.body.includedKeyword.every((el) => isValidString(el))
+  ) {
+    return res.status(400).send({ message: "[InvalidIncludedKeyword] Error occured" });
+  }
+
+  if (
+    !isValidArray(req.body.excludedKeyword) ||
+    !req.body.excludedKeyword.every((el) => isValidString(el))
+  ) {
+    return res.status(400).send({ message: "[InvalidExcludedKeyword] Error occured" });
+  }
+
+  if (!isValidString(req.body.ownerUid) || isEmptyString(req.body.ownerUid)) {
+    return res.status(400).send({ message: "[InvalidOwnerUid] Error occured" });
+  }
+
+  try {
+    const { keywordId } = req.params;
```

Comment:
```suggestion
    const keywordId = req.params.keywordId;
```
`params`에서 `keywordId`만 가져오는 것이기 때문에, 구조분해할당하지 않고 바로 변수에 할당하는 게 더 좋을 것 같습니다.